homeassistant/components/freebox/__init__.py

```diff
@@ -28,7 +28,7 @@
     extra=vol.ALLOW_EXTRA,
 )
 
-SCAN_INTERVAL = timedelta(seconds=30)
+SCAN_INTERVAL = timedelta(seconds=5)
```

Comment:
5 sec of scan interval is very low, why ?

Reply:
Today I get Freebox’s sensors using Node-Red and MQTT. The polling interval is set at 5 secs and it’s working fine.
Using a lower poolling frequency will cause somme issues if the sensors are used with an alarm system

Reply:
Oh, so now that I checked, I think I know why there are error catching everywhere : `asyncio.TimeoutError`, native `TimeoutError` and `aiohttp.client_exceptions.ClientError`.
It's because of a 5s scan_interval !

With 5s, you are stressing the router (for nothing), and that's why you are having timeouts !!
Event 30s is sometimes short: in a other integration I used to developped, `synology_dsm`, which are NAS, even some models needed 45s+ !

It's working fine because your sensor don't get updated few times in an hour (or even minute) but you don't even notice it ?
Because you catch all cause of issues you created when reducing the scan_interval ?

It should not work like this (catch every error everywhere, by default, without reason).

So:
1. Remove all catch of timeout/client related errors
2. scan_interval should be at least 15s, as a bare minimum, 30 is nice, I don't think it should be changed, or add a option flow to configure a custom scan_interval (but in a new PR).

I prefer using `assumed_state = True` on specific (binary) sensors entities than reducing the scan_interval for everyone, even people which don't use Home sensors !
Here is the related doc: https://developers.home-assistant.io/docs/core/entity/

Reply:
SCAN_INTERVAL set to 30s